.changeset/hungry-apples-sparkle.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/catalog-model': patch
+---
+
+Deprecates the `EntityRefContext` type which had limited use.
```

Comment:
Ah realizing this wasn't a typo in the other PR x)

I'm thinking the way you'd read this if you include the version number/package when reading the changelog would be "catalog-model 0.3.4 deprecated the ..." rather than "catalog-model 0.3.4 deprecates the ...". As in let's use past tense? 😅 Not entirely sure though